src/translation/translation_nl_NL.ts

```diff
@@ -4008,7 +4008,7 @@ We hebben uw kanaal gedempt en &apos;Demp mijzelf&apos; geactiveerd. Los eerst h
     <message>
         <location filename="../serverdlg.cpp" line="737"/>
         <source>Now a directory server</source>
```

Comment:
Sorry, as I mentioned in the French translation, can you avoid "server" where you see "directory server" in English - the English needs updating.

Reply:
The only reason why I include "server" is because the English text contains it. WIll update the PR.

Reply:
@pljones only found two strings that needed updating. 

The strings which still include "server" are marked as vanished, so have not update those.

Reply:
Good, sounds like the Dutch has been kept up to date with the English, then.  I think you can delete anything marked `"vanished"` (but someone else should confirm).

Reply:
> anything marked `"vanished"` (but someone else should confirm).

I agree, "vanished" should be possible to be removed. I even believe we should look at cleaning up the .ts files post release, so that during the release the translator can make use of the vanished strings (in case needed) and then after the release a cleanup (script) is run to remove the vanished strings from all langauge files. After all this only requires to use an extra option with the lupdate command (-noobsolete) to have them removed.
Or instead of post release, it can be part of the create release process as well.

Reply:
> can you avoid "server" where you see "directory server" in English - the English needs updating

Changing translations to make them "correct" before the English version has been corrected is the wrong way to go about it. When the English text is finally updated, those strings will be marked as "untranslated" in the translations, making it necessary to edit them yet again anyway. It's better to merge #2709 now and then tell translators to update their files.

Reply:
Ah, right - I didn't realise that.  Thanks!